inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/PostgreSqlSource.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.agent.plugin.sources;
+
+import org.apache.inlong.agent.conf.JobProfile;
+import org.apache.inlong.agent.plugin.Reader;
+import org.apache.inlong.agent.plugin.Source;
+import org.apache.inlong.agent.plugin.sources.reader.PostgreSqlReader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.apache.inlong.agent.constant.CommonConstants.PROXY_INLONG_GROUP_ID;
+import static org.apache.inlong.agent.constant.CommonConstants.DEFAULT_PROXY_INLONG_GROUP_ID;
+import static org.apache.inlong.agent.constant.CommonConstants.PROXY_INLONG_STREAM_ID;
+import static org.apache.inlong.agent.constant.CommonConstants.DEFAULT_PROXY_INLONG_STREAM_ID;
+import static org.apache.inlong.agent.constant.AgentConstants.GLOBAL_METRICS;
+
+/**
+ * PostgreSql source, split PostgreSql source job into multi readers
+ */
+public class PostgreSqlSource implements Source {
```

Comment:
Suggest changing the class name to `PostgreSQLReader`.

Reply:
Do you mean `PostgreSQLSource`, beacause this class implements `Source` interface

Reply:
`PostgreSQLSource` looks good to me. Maybe we can `extends AbstractSource` instead of `implements Source` for referring to `KafkaSource` and `BinlogSource`?

Reply:
Sorry I didn't describe it clearly.

My concern is that `PostgreSQL` is the official name, and `Postgres`, `PostgreSql`, etc. are not official names.

Reply:
OK, fixed